spec/system/user/view_user_index_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
 
-RSpec.describe "User index", type: :system do
+RSpec.describe "User index", type: :system, stub_elasticsearch: true do
```

Comment:
I am REALLY happy with this change! I can promise you that future me in 6 months will know exactly what this flag is and when to use it :)